### PR TITLE
Added bucket name to s3 drone plugin

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by lafriks** [comment]
LGTM

**Comment by lunny** [comment]
but the `s3_access_key, s3_secret_key` seems changed?

File: .drone.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ pipeline:
     image: plugins/s3:1
     pull: true
     secrets: [ s3_access_key, s3_secret_key ]
```
Comment (lunny) [inline .drone.yml]: seems changed in another PR.

**Review by lunny — CHANGES_REQUESTED** [review]

**Comment by tboerger** [comment]
> but the s3_access_key, s3_secret_key seems changed?

Right, good catch.

**Review by lunny — APPROVED** [review]

**Pull request merged**